### ISSUE TITLE
`--breaks encoded` not working without sb elements in Verovio 3.6

### ISSUE THREAD

**Issue by wergo** [rism-digital/verovio#2358]
With the current release 3.6, the `--breaks` option seems to have changed its behavior.

This dummy encoding containing three `<pb>` elements, but no `<sb>` elements appears incorrectly on one page, when called on the command line `verovio --breaks encoded -a pagebreaks.mei`. It should (and it has in previous versions) appear on three pages. (Similarly, `--breaks line` results in the same single-page output.)

<img width="504" alt="grafik" src="https://user-images.githubusercontent.com/45632600/130739175-38141ed6-7ec7-47c7-bc5a-885415de6e22.png">

<details><summary>pagebreaks.mei with three page break elements</summary> 

```XML
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title>Three page breaks</title>
            <respStmt />
         </titleStmt>
         <pubStmt />
      </fileDesc>
   </meiHead>
   <music>
      <body>
         <mdiv>
            <score>
               <scoreDef key.sig="0" meter.count="3" meter.unit="4">
                  <staffGrp bar.thru="true" symbol="brace">
                     <staffDef n="1" lines="5" clef.shape="G" clef.line="2" />
                     <staffDef n="2" lines="5" clef.shape="F" clef.line="4" />
                  </staffGrp>
               </scoreDef>
               <section>
                  <pb />
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="4" pname="a" />
                           <note dur="4" oct="4" pname="b" />
                           <note dur="4" oct="5" pname="c" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="5" pname="d" />
                           <note dur="4" oct="5" pname="c" />
                           <note dur="4" oct="4" pname="b" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <pb />
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="4" pname="a" />
                           <note dur="4" oct="4" pname="g" />
                           <note dur="4" oct="4" pname="f" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="4" pname="e" />
                           <note dur="4" oct="4" pname="d" />
                           <note dur="4" oct="4" pname="c" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <pb />
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="3" pname="b" />
                           <note dur="4" oct="4" pname="c" />
                           <note dur="4" oct="4" pname="d" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="4" pname="e" />
                           <note dur="4" oct="4" pname="f" />
                           <note dur="4" oct="4" pname="g" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
               </section>
            </score>
         </mdiv>
      </body>
   </music>
</mei>
```
</details>

It works as expected on three pages when also `<sb>` elements are present in the encoding: 

<img width="245" alt="grafik" src="https://user-images.githubusercontent.com/45632600/130739616-8679f1bd-c749-42bf-9c77-fc69f5d928c0.png"><img width="245" alt="grafik" src="https://user-images.githubusercontent.com/45632600/130739376-280b4553-f252-4365-a33b-c6acb2410645.png"><img width="245" alt="grafik" src="https://user-images.githubusercontent.com/45632600/130739401-2cf94c8a-4ddb-443c-89a8-15544cdce9d1.png">

<details><summary>pagesystembreaks.mei with three page breaks and three system breaks</summary> 

```XML
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title>Three page and three system breaks</title>
            <respStmt />
         </titleStmt>
         <pubStmt />
      </fileDesc>
   </meiHead>
   <music>
      <body>
         <mdiv>
            <score>
               <scoreDef key.sig="0" meter.count="3" meter.unit="4">
                  <staffGrp bar.thru="true" symbol="brace">
                     <staffDef n="1" lines="5" clef.shape="G" clef.line="2" />
                     <staffDef n="2" lines="5" clef.shape="F" clef.line="4" />
                  </staffGrp>
               </scoreDef>
               <section>
                  <pb />
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="4" pname="a" />
                           <note dur="4" oct="4" pname="b" />
                           <note dur="4" oct="5" pname="c" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <sb />
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="5" pname="d" />
                           <note dur="4" oct="5" pname="c" />
                           <note dur="4" oct="4" pname="b" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <pb />
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="4" pname="a" />
                           <note dur="4" oct="4" pname="g" />
                           <note dur="4" oct="4" pname="f" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <sb />
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="4" pname="e" />
                           <note dur="4" oct="4" pname="d" />
                           <note dur="4" oct="4" pname="c" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <pb />
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="3" pname="b" />
                           <note dur="4" oct="4" pname="c" />
                           <note dur="4" oct="4" pname="d" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
                  <sb />
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <note dur="4" oct="4" pname="e" />
                           <note dur="4" oct="4" pname="f" />
                           <note dur="4" oct="4" pname="g" />
                        </layer>
                     </staff>
                     <staff n="2">
                        <layer n="1">
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                           <note dur="4" oct="3" pname="a" />
                        </layer>
                     </staff>
                  </measure>
               </section>
            </score>
         </mdiv>
      </body>
   </music>
</mei>
```
</details>

I have not tracked down the commit where the behavior changed. Help highly appreciated.

**Comment by craigsapp** [comment]
Yes, it is a problem and I had to disable page-view mode for now in VHV because of it.

Some related issues and discussion:

https://github.com/rism-digital/verovio/issues/2034
https://github.com/rism-digital/verovio/issues/1987

The problem was generated when `--breaks smart` were implemented:

https://github.com/rism-digital/verovio/issues/1931

**Comment by wergo** [comment]
The page break option is essential for the speedmode of mei-friend and it used to work fine with Verovio release 3.4.1, works with unreliable error messages in the tk.render2svg in 3.5.0, but stopped working properly in 3.6. So, the problems are related, but this specific problem was introduced very recently.

**Comment by lpugin** [comment]
Yes, I suspect I introduced the problem when implementing the multiple `mdiv/score` feature. I'll have a look.

**Comment by lpugin** [comment]
Should be fixed with 50819d83f1f6707ea019478689e76d57924057a8
